### ISSUE TITLE
Too many false positivies

### ISSUE THREAD

**Issue by fulldecent** [endymion/link-checker#17]
The program is not reliable because it produces many false positives. I need something reliable that I can use in CI.

When reviewing: https://www.acls.net/rhode-island-acls-pals-bls.htm

However, by spot checking these links in Chrome I can confirm that most do work and are not blocked or anything by robots.txt. Even `curl` works on these pages.

Please advise.

```
Checked: ./oregon-acls-pals-bls.htm
Problem: ./rhode-island-acls-pals-bls.htm
   Link: http://www.providencejournal.com/breaking-news/content/
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.wpri.com/dpp/news/local_news/east_bay/man-dies-after-barrington-sailing-race
     Response: #<Net::HTTPNotFound:0x007fea3b0ce450>
Problem: ./new-york-acls-pals-bls.htm
   Link: http://www.medpagetoday.com/Endocrinology/Obesity/40920
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.nytimes.com/2011/08/08/sports/man-dies-during-new-york-city-triathlon.html?_r=1
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.nytimes.com/2009/11/02/sports/02marathonbox.html?_r=2
     Response: getaddrinfo: nodename nor servname provided, or not known
Problem: ./ca-san-diego.htm
   Link: http://www.ecsinstitute.org/Resources/ECSI_CAAlgorithm.pdf
     Response: getaddrinfo: nodename nor servname provided, or not known
Problem: ./a-history-of-heart-transplants.htm
   Link: https://www.fi.edu/learn/heart/history/firsts.html
     Response: #<Net::HTTPNotFound:0x007fea3c28d510>
   Link: http://www.pbs.org/wgbh/aso/databank/entries/bmbarn.html
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://stanfordhospital.org/cardiovascularhealth/cardiacSurgeryTimeline.html
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.sahistory.org.za/dated-event/dr-chris-barnard-performs-worlds-first-human-heart-transplant
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.tshaonline.org/handbook/online/articles/sdh01
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.westchestermedicalcenter.com/TheHistoryofHeartTransplants
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://newsroom.ucla.edu/portal/ucla/man-s-heart-removed-and-replaced-241430.aspx
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.llu.edu/news/babyfae/infant-transplant-timeline.page
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.transplanthouse.org/
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.ishlt.org
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://web.clark.edu/lbramson/links/d.html
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://optn.transplant.hrsa.gov/governance/about-the-optn/
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.health.wyo.gov/phsd/heartdisease/links.html
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://nycardiothoracic.org/
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.ohsu.edu/xd/health/services/heart-vascular/services/heart-failure-transplant/
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.nyu.edu/clubs/organ.donation/links.htm
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.supportcircle.com
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.heart-transplant.co.uk
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.donatelifeny.org
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.trioweb.org
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://hearttransplant.com
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://lomalindahealth.org/medical-center/our-services/transplantation/services-and-procedures/heart-transplant/index.page
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.emoryhealthcare.org/
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://www.heartonline.org/heart_failure.htm
     Response: getaddrinfo: nodename nor servname provided, or not known
   Link: http://saturn.ece.ndsu.nodak.edu/ecewiki/index.php/Heart_Transplant
     Response: No route to host - connect(2)
Problem: ./ca-san-francisco.htm
   Link: http://www.navy.mil/submit/display.asp?story_id=20550
     Response: No route to host - connect(2)
   Link: http://depts.washington.edu/learnCPR/quickCPR.html
```

**Comment by karlosmid** [comment]
Hi @fulldecent , please check that you/script have rights to execute gedaddrinfo in folder where it is run.

**Comment by fulldecent** [comment]
OK, I see what happening. My DNS here sucks. I can't blame this on your software. Closing.
